include/picongpu/particles/ionization/byField/IonizationCurrent/IonizationCurrent.hpp

```diff
@@ -58,11 +58,16 @@ namespace ionization
         HDINLINE void
         operator()(IonizerReturn retValue, float_X const weighting, T_JBox jBoxPar, ValueType_E eField, T_Acc const & acc, floatD_X const pos)
         {
-            auto ionizationEnergy = weighting * retValue.ionizationEnergy * SI::ATOMIC_UNIT_ENERGY / UNIT_ENERGY; // convert to PIConGPU units
-            /* calculate ionization current at particle position */
-            float3_X jIonizationPar = JIonizationCalc{}(ionizationEnergy, eField);
-            /* assign ionization current to grid points */
-            JIonizationAssignment<T_Acc, T_DestSpecies, simDim>{}(acc, jIonizationPar, pos, jBoxPar);
+            if(retValue.ionizationEnergy == 0)
```

Comment:
Could you please write `if(retValue.ionizationEnergy != 0.0_X)`. This will avoid the empty body.